src/test/java/org/folio/rest/impl/RestVerticleTest.java

```diff
@@ -83,36 +81,35 @@ public void setUp(TestContext context) throws IOException {
           .put("totalRecords", 0).encode())));
 
     Async async = context.async();
-    TenantClient tenantClient = new TenantClient("localhost", okapiPort, "diku", "diku");
-    vertx = Vertx.vertx();
     okapiUrl = "http://localhost:" + okapiPort;
     templateUrl = okapiUrl + "/templates";
+    TenantClient tenantClient = new TenantClient(okapiUrl, "diku", null);
+    vertx = Vertx.vertx();
+
+    PostgresClient.getInstance(vertx).startEmbeddedPostgres();
 
     DeploymentOptions options = new DeploymentOptions().setConfig(
       new JsonObject()
         .put("http.port", okapiPort)
     );
-    try {
-      PostgresClient.setIsEmbedded(true);
-      PostgresClient.getInstance(vertx).startEmbeddedPostgres();
-    } catch (Exception e) {
-      e.printStackTrace();
-      context.fail(e);
-      return;
-    }
+
     vertx.deployVerticle(RestVerticle.class.getName(), options, res -> {
       try {
-        tenantClient.postTenant(null, res2 -> {
+        TenantAttributes t = new TenantAttributes().withModuleTo("mod-template-engine-1.0.0");
+        tenantClient.postTenant(t, res2 -> {
           async.complete();
         });
       } catch (Exception e) {
-        e.printStackTrace();
+        logger.error(e.getMessage());
```

Comment:
This should be replaced by `context.fail(e)`. JUnit will then stop testing and report the message and stacktracae of e.

Reply:
Replaced